quad_utils/rviz/example_with_terrain.rviz

```diff
@@ -1,6 +1,5 @@
 Panels:
   - Class: rviz/Displays
-    Help Height: 0
```

Comment:
I still see the (unpublished) Estimated State in the RViz view, can we revert that so it doesn't show up by default?